src/components/AdherenceCalendar.js

```diff
@@ -56,7 +56,7 @@ const AdherenceCalendar = observer(({ store }) => (
           date={date}
           medication_report={
             view === 'month' &&
-              store.medication_reports.records.find(mr =>
+              store.registration.information.medication_reports.find(mr =>
```

Comment:
New way to access data!
More reliable!
More appropriate for how we're using it in the UI!